docs/devs/dev.md

```diff
@@ -4,8 +4,10 @@
 Dev setup can be found in the original Synthea [instructions](https://github.com/synthetichealth/synthea/wiki/Developer-Setup-and-Running).
 
 ## Development types
-There are two types of development for the code:
+There are several types of development for the code:
 
 - To develop the underlying engine see java.md.
 
-- To develop the underlying information that the simulation works on read the [resources page](resources.md).
\ No newline at end of file
+- To develop the underlying information that the simulation works on read the [resources page](resources.md).
+
+- To create new disease modules read the [module creation page](modules.md)
```

Comment:
I asked chat gpt to rephrase this: 

The code can be developed in several ways:

To develop the underlying engine, refer to java.md.
To update the foundational data driving the simulation, see the [resources page](https://github.com/nhsengland/swpc_synthea/pull/32/resources.md).
To design new disease modules, check out the [module creation page](https://github.com/nhsengland/swpc_synthea/pull/32/modules.md).